graphql_api/types/owner/owner.graphql

```diff
@@ -1,5 +1,5 @@
 type Owner {
-  avatarUrl: String
+  avatarUrl: String!
```

Comment:
I see a lot of logic going on in the avatar_url field, just confirming you went through it all and no children resort to None

Reply:
Yep tracing through the `Owner.avatar_url` function, every code path should result in a string being returned.